app/components/utils/agendaitem-table.js

```diff
@@ -100,25 +100,28 @@ export default class AgendaitemTable extends Component {
       sortColumn.set('sorted', true);
     }
 
-    this.setRowsPostponed(table.rows, this.model);
+    this.setRowsNotApproved(table.rows, this.model);
     return table;
   }
 
   /**
-   * Will set the postponed classes on all actual table rows that should get the postponed class
+   * Will set the greyed classes on all actual table rows that have postponed or retracted decisions
    * It does so based on the current model and table rows that you supply
    * @param {Table} tableRows       Table object containing rows
    * @param {EmberArray} model      Model from the route currently active used as a lookup
    */
-  setRowsPostponed(tableRows, model) {
+  setRowsNotApproved(tableRows, model) {
     const rowsCurrentlyInTable = tableRows;
-    const postponedItems = model.filter((agendaitem) => agendaitem.get('retracted'));
-    postponedItems.forEach((postponedItem) => {
-      const postponedRowInTable = rowsCurrentlyInTable.find(
-        (rowFromTable) => rowFromTable.content.get('id') === postponedItem.get('id')
+    // relations are included in route
+    const postponedItems = model.filter((agendaitem) => agendaitem.get('treatment.decisionActivity.decisionResultCode.isPostponed'));
+    const retractedItems = model.filter((agendaitem) => agendaitem.get('treatment.decisionActivity.decisionResultCode.isRetracted'));
```

Comment:
Filtering an array on an async property will not work if the data isn't fetched already. But since this component is only used in the print-overviews, which are currently not actively used in the app and require a major rewrite, I would leave it as it is in this story.

Reply:
It works, the data should be fetched with `includes` from  frontend-kaleidos/app/routes/print-overviews/decisions/agendaitems.js

Reply:
Yes, I know. But even then we have the convention of still explicitely awaiting the relationship (even though we know that the promise will immediately resolve) because someone might remove the include in the future and not be aware that this will break things here.

But as said, since this component requires a major rewrite, you can just leave it as it is for now.